libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -143,6 +144,14 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO_FRAME("OPTIONS", 9,   AP_Arming, _arming_options, 0, AP_PARAM_FRAME_COPTER),
 
+    // @Param: WPDST_MAX
+    // @DisplayName: Maximum allowed distance to first waypoint.
```

Comment:
Ideally we should start the display name with "Arming".  So perhaps, "Arming max distance to first waypoint".